lib/govuk_app_config/govuk_content_security_policy.rb

```diff
@@ -11,7 +11,8 @@ module GovukContentSecurityPolicy
   # - https://csp-evaluator.withgoogle.com
   # - https://cspvalidator.org
 
-  GOVUK_DOMAINS = "'self' *.publishing.service.gov.uk localhost".freeze
+  PROD_GOVUK_DOMAINS = "'self' *.publishing.service.gov.uk localhost".freeze
```

Comment:
Now that `GOVUK_DOMAINS` is broken into dev and prod. Is `localhost` needed in production?

Reply:
Good point. I can't think of a realistic scenario for running in prod on localhost (i.e. dev machine), since a bunch of other stuff usually breaks when you try that anyway.

Reply:
I've added a commit to remove it in prod.

Reply:
@rubenarakelyan is there reason for having `localhost` in production?

Reply:
I was asking Ruben about that yesterday. I think it should be fine to go as I think it expects to connect to localhost over https and I doubt that happens. Ruben thought it might be needed as part of tests but wasn't sure.

So worth having a go at killing it, may try running this branch against government-frontend if you want to be extra cautious?